xonsh/completers/tools.py

```diff
@@ -0,0 +1,33 @@
+import builtins
+
+
+def get_filter_function():
```

Comment:
It's the function that is used to `filter` the list, was my reason for calling it that.  Agree about defining them globally.  Will make that change.